recipe/meta.yaml

```diff
@@ -39,33 +39,36 @@ requirements:
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
     - m2-patch  # [py==310 and win]
+    
 
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.39.*
+    - llvmlite =0.40.0
     - numpy {{ numpy_min_ver }}
-    - tbb-devel 2021.*,<2021.6
+    - tbb-devel >=2021.6
     - _openmp_mutex            # [linux]
     - importlib_metadata       # [py<39]
 
   run:
     - python
     - importlib_metadata       # [py<39]
-    - llvmlite >=0.39.*,<0.40
+    - llvmlite >=0.40
```

Comment:
I think it's better to constrain as `llvmlite >=0.40.0,<0.41.0a0`, see https://github.com/numba/numba/blob/4fd4e39c672d119b54a2276d170f270764d2bce7/setup.py#L27-L28

Reply:
done